_docs/sysadmin/system_customization.md

```diff
@@ -32,64 +45,74 @@ the command is:
 sudo ./bin/setcsvfields 13 12 15 7
 ```
 
+<small>[Back To Table of Contents](#toc)</small>
 
 
-### Schedule backups of production server data
+### <a name="schedule_backups">Schedule backups of production server data</a>
 
 Specifically, the configuration, submission, and results data for all courses:
 
 ```
 /var/local/submitty/courses/
 ```
 
-And the central location of the student VCS (git/svn/etc. version control) repositories (if used):
+And the central location of the student VCS (e.g. git version control) repositories (if used):
```

Comment:
we can probably just shorten this whole thing to just be `(e.g. git)` and have it read a bit better/naturally.